src/encoded/static/components/matrix_experiment.js

```diff
@@ -106,6 +106,7 @@ const convertExperimentToDataTable = (context, getRowCategories, getRowSubCatego
         colTitleMap[colCategoryBucket.key] = colIndex;
         return colCategoryBucket.key;
     });
+    const searchUrl = '/search/?type=Experiment&status=released';
```

Comment:
I feel uncomfortable hard-coding this. I’d prefer to do something like taking the search_base and removing any part of it that matches a column property before appending that column property.